src/modules/wreck/job.c

```diff
@@ -180,6 +180,7 @@ static flux_future_t *send_create_event (flux_t *h, struct wreck_job *job)
                              "ntasks", job->ntasks,
                              "ncores", job->ncores,
                              "nnodes", job->nnodes,
+                             "ngpus", job->ngpus,
```

Comment:
The extra `s:i` parameter for the `"ngpus", job->gpus` pair is missing in the `flux_msg_pack()` format here.